### PR TITLE
[v6r15] Truly minor fixes while looking around

### PR DESCRIPTION
**Pull request by fstagni**

### CODE REVIEW

**Comment by chaen** [comment]
review OK

File: DataManagementSystem/scripts/dirac-dms-filecatalog-cli.py [hunk 1]
```diff
@@ -1,11 +1,10 @@
 #!/usr/bin/env python
-########################################################################
-# $HeadURL$
-########################################################################
-__RCSID__ = "$Id$"
+
+import sys
+
+from DIRAC import gConfig, exit as dexit
```
Comment (andresailer) [inline DataManagementSystem/scripts/dirac-dms-filecatalog-cli.py]: Is it really save (for all eternity) to put this gConfig import before Script.parseCommandLine is called?
(Is there a rule for what can be imported before parseCommandLine ?)

Comment (chaen) [inline DataManagementSystem/scripts/dirac-dms-filecatalog-cli.py]: well spoted ! I missed that one. In principe the rule (written somewhere) is that parseCommandLine should be the very first thing. So indeed, gConfig should be imported later

Comment (fstagni) [inline DataManagementSystem/scripts/dirac-dms-filecatalog-cli.py]: Thanks, corrected

**Pull request merged**